psychopy/visual/movie3.py

```diff
@@ -312,7 +312,13 @@ def _updateFrameTexture(self):
             if self._nextFrameT > (self._videoClock.getTime() -
                                    self._retraceInterval/2.0):
                 return None
-        self._numpyFrame = self._mov.get_frame(self._nextFrameT)
+        try:
+            self._numpyFrame = self._mov.get_frame(self._nextFrameT) 
+        except:
```

Comment:
Rather than catching all errors here we need to catch only the one that's relevant (otherwise, when other errors get swallowed here it's hard to debug why). So line 317 should be:
```
        except OSError:
```